packages/cactus-common/src/test/typescript/unit/key-converter.test.ts

```diff
@@ -0,0 +1,395 @@
+import test, { Test } from "tape";
+
+import crypto from "crypto";
+import KeyEncoder from "key-encoder";
+import secp256k1 from "secp256k1";
+
+import { JsObjectSigner } from "../../../main/typescript/public-api";
+
+import {
+  KeyConverter,
+  KeyFormat,
+} from "../../../main/typescript/key-converter";
+
+const keyConverter = new KeyConverter();
+
+let privKey: any;
+// generate secp256K1 private key
+do {
+  privKey = crypto.randomBytes(32);
+} while (!secp256k1.privateKeyVerify(privKey));
+
+// generate secp256K1 public key
+const pubKey = secp256k1.publicKeyCreate(privKey);
+
+const keyPairBuffer = { privateKey: privKey, publicKey: pubKey };
+
+const keyEncoder: KeyEncoder = new KeyEncoder("secp256k1");
+const hexPublic = Buffer.from(pubKey).toString("hex");
+const pemPublic = keyEncoder.encodePublic(
+  Buffer.from(pubKey).toString("hex"),
+  "raw",
+  "pem"
+);
+const hexPrivate = privKey.toString("hex");
+const pemPrivate = keyEncoder.encodePrivate(
+  privKey.toString("hex"),
+  "raw",
```

Comment:
I thought we should have more variants here. 
Int8Buffer, CryptoKey, encrypted key, ...